tools/tfm/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2017-2018 ARM Limited
```

Comment:
This file is identical: https://github.com/ARMmbed/mbed-os/blob/master/tools/spm/__init__.py